lib/ansible/modules/files/find.py

```diff
@@ -107,6 +107,22 @@
               to false will override this value, which is effectively depth 1.
               Default is unlimited depth.
         version_added: "2.6"
+    noxfs:
+        description:
+            - If true, filesystem boundaries will not be crossed.
+        type: bool
+        default: False
+        version_added: "2.8"
+    noxfs_overrides:
```

Comment:
Im fine with noxfs, the noxfs_overrides seems too much, a user can just have 2 finds with diff noxfs at that point